charts/security-apps/values.yaml

```diff
@@ -13,7 +13,7 @@ dex:
   # -- Chart
   chart: "dex"
   # -- [dex Helm chart](https://github.com/dexidp/helm-charts/tree/master/charts/dex/) version
-  targetRevision: "0.14.0"
+  targetRevision: "0.14.2"
```

Comment:
Why the double bump?

Reply:
I think I've missed the update to 0.14.1.